### ISSUE TITLE
rw_benchmark doesn't seem to exist

### ISSUE THREAD

**Issue by sgpthomas** [NYU-NEWS/janus#4]
Both `config/rw.yml` and `config/rw_fixed.yml` have reference to `rw_benchmark` which doesn't seem to exist. This can be fixed by replacing `rw_benchmark` with `rw`. There is also a reference to `rw_benchmark` in the readme. 

```
I [s_main.cc:122] 2017-06-08 12:21:22.709 | PWD : 
I [s_main.cc:134] 2017-06-08 12:21:22.709 | starting process 2386
I [config.cc:268] 2017-06-08 12:21:22.709 | LoadYML: config/tapir.yml
I [config.cc:268] 2017-06-08 12:21:22.710 | LoadYML: config/2c2s1p.yml
I [config.cc:383] 2017-06-08 12:21:22.712 | BuildSiteProcMap
I [config.cc:268] 2017-06-08 12:21:22.712 | LoadYML: config/rw.yml
E [config.cc:462] 2017-06-08 12:21:22.713 | No implementation for benchmark: rw_benchmark
  *** begin stack trace ***
0    rrr::print_stack_trace(_IO_FILE*)                                                                              /users/sgt43/janus/build/../rrr/base/debugging.cpp:65
1    rococo::Config::InitBench(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&)    /users/sgt43/janus/build/../deptran/config.cc:463
2    rococo::Config::LoadBenchYML(YAML::Node)                                                                       /users/sgt43/janus/build/../deptran/config.cc:514
3    boost::detail::shared_count::~shared_count()                                                                   /usr/include/boost/smart_ptr/detail/shared_count.hpp:443
4    rococo::Config::Load()                                                                                         /users/sgt43/janus/build/../deptran/config.cc:255
5    rococo::Config::CreateConfig(int, char**)                                                                      /users/sgt43/janus/build/../deptran/config.cc:198
6    main                                                                                                           /users/sgt43/janus/build/../deptran/s_main.cc:138
7    /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0) [0x7ff89a38b830]
  ***  end stack trace  ***
  *** verify failed: 0 at ../deptran/config.cc, line 463
```

**Comment by shuaimu** [comment]
Thanks for the catch! There were some inconsistencies among the benchmark names. I will fix it.

@lamontnelson Do you remember which bench we actually used for single key tests? Is it "micro" or "rw"?

**Comment by lamontnelson** [comment]
I believe we used "rw".

**Comment by shuaimu** [comment]
fixed in commit 4e374c3
